iothub_client/samples/iothub_client_sample_upload_to_blob/iothub_client_sample_upload_to_blob.c

```diff
@@ -6,7 +6,7 @@
 // when writing production code.
 
 #ifdef DONT_USE_UPLOADTOBLOB
-#error "trying to compile iothub_client_sample_upload_to_blob.c while DONT_USE_UPLOADTOBLOB is #define'd"
+// "trying to compile iothub_client_sample_upload_to_blob.c while DONT_USE_UPLOADTOBLOB is #define'd"
```

Comment:
Do we need this comment here?  Does it do anything?

Reply:
Oops. Removing before the push.